INSTALL.md

```diff
@@ -0,0 +1,101 @@
+# From new CentOS 7 install:
+
+## Disable selinux:
+### change `/etc/selinux/config` to `SELINUX=disabled`
+
+## Install Postgreql 9.6 client and libraries
+
+Instructions are here: https://yum.postgresql.org/
+
+- grab the link for CentOS 7 and install:
+    $ sudo yum install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-7-x86_64/pgdg-centos96-9.6-xxxx.noarch.rpm
+
+- install `postgresql96` (for psql commands) and `postgresql96-devel` (for includes/libraries needed to install `DBD::Pg` perl library)
+    $ sudo yum install postgresql96 postgresql96-devel
+
+## Install Postgres 9.6 server (in a container or on the host)
+
+### on the host:
+   $ sudo yum install postgresql96-server
+
+### -or- in a container
+
+NOTE: you do *not* need postgresql96-server if running postgres within a `docker` container.
+
+Install `docker` and `docker-compose` using instructions here:
+    https://docs.docker.com/engine/installation/linux/centos/
+    https://docs.docker.com/compose/install/
+
+Add yourself to `docker` group
+    $ sudo usermod -G docker $USER
+
+Remember to logout and login again..   You should see `docker` in your list of groups:
+
+    $ id
+    uid=9876(myuser) gid=9876(myuser) groups=9876(myuser),990(docker) ...
+
+Start a docker container to run postgres
+    $ cd incubator-trafficcontrol/traffic_ops/app/db/pg-migration
+    $ ./start_postgres.sh
+
+Edit `mysql-to-postgres.yml` to suit your needs.
+- modify `POSTGRES_*` vars to apply to new postgres container that will house
+
+If migrating from an existing pre-2.0 traffic_ops server (mysql):
+- `TO_*` vars for admin access to existing mysql-based `traffic_ops` (to get a db dump)
+- `MYSQL_*` vars to apply to temporary mysql container
+
+Run migration from existing mysql-based `traffic_ops`
+    $ ./migrate.sh
+
+## Install `traffic_ops`
+    $ sudo yum install traffic_ops
+
+## Install `go` and `gcc` (required for `goose` and some `Perl` modules)
+    $ sudo yum install go gcc
+    
+## Install Perl modules
+    $ sudo cpanm Carton
+
+IMPORTANT!!: We're using a later version of Postgresql,  so it's not installed in the default place.
+We need to tell carton where it is so the `DBD::Pg` module is installed correctly.
+
+    $ sudo su -
+    # cd /opt/traffic_ops/app
+    # POSTGRES_HOME=/usr/pgsql-9.6 /usr/local/bin/carton
+
+
+## Install goose
+    $ sudo GOPATH=/tmp GOBIN=/usr/local/bin go get bitbucket.org/liamstask/goose/cmd/goose
+
+
+## Modify `traffic_ops` configuration
+
```

Comment:
@dangogh Examples of values, even if generic but following the expected convention, for common entries to update would be a useful addition for those new to the project for the following section's configuration files.